tests/interpretation/test_gaussian_probe.py

```diff
@@ -0,0 +1,30 @@
+import random
+import unittest
+
+import numpy as np
+import torch
+
+from neurox.interpretation.gaussian_probe import get_neuron_ordering, train_probe
+
+
+class TestGetNeuronOrdering(unittest.TestCase):
+    def test_get_neuron_ordering(self):
+        "Basic get neuron ordering test"
+
+        # The neuron 0 is perfectly correlated with label, the neuron 1 is partly correlated
+        # The neuron 2 is not correlated
+
+        X = np.array(
+            [
+                [-1.1, -1.05, 0.01],
+                [-1.05, 0, 0.005],
+                [1.1, 1.05, 0.003],
+                [1.03, 0, 0.1],
+            ],
+            dtype=float,
+        )
+        y = np.array([0, 0, 1, 1])
+        expected_neuron_order = [0, 1, 2]
```

Comment:
Can you rearrange the data such that the neuron order is _not_ `0,1,2`; in case the implementation always picks the "next available neuron", this test will not distinguish it from a working implementation;